omnibus/files/private-chef-cookbooks/private-chef/templates/default/nginx/nginx_chef_api_lb.conf.erb

```diff
@@ -44,7 +44,11 @@
       default_type text/plain;
       alias /opt/opscode/version-manifest.txt;
     }
-
+    # location for Let's Encrypt challenges
+    location /.well-known/acme-challenge {
```

Comment:
Can we make this opt-in? How about adding a configuration flag?

If you add 

    default['private_chef']['nginx']['lets_encrypt_enable'] = false

[here](https://github.com/chef/chef-server/blob/7b31228c49ac9e45b50c43e07b461e343cf1c71e/omnibus/files/private-chef-cookbooks/private-chef/attributes/default.rb#L489), and wrap that bit in

    <% if node['private_chef']['nginx']['lets_encrypt_enable'] -%>
    # location for Let's Encrypt challenges
    ...
    <% end -%>

only users that add

    nginx['lets_encrypt_enable'] = true

to `chef-server.rb` would have this config bit rendered.

There might be better keys than `lets_encrypt_enable`, but you get the drift.

Also, I'm not sure if we'd want to use a special, separate location for that challenge data? Maybe not serve it out of the ordinary webroot? (I'm not sure right now if it's used for anything else...)